workshop/main.rb

```diff
@@ -0,0 +1,50 @@
+require_relative "./selenium.rb"
+class LoginFlow
```

Comment:
The class shouldn't be named LoginFlow as it contains the `search` flow too. Rename appropriately